lib/experiments-page.php

```diff
@@ -138,6 +138,18 @@ function gutenberg_initialize_experiments_settings() {
 		)
 	);
 
+	add_settings_field(
+		'gutenberg-pattern-partial-syncing',
+		__( 'Synced patterns partial syncing', 'gutenberg' ),
+		'gutenberg_display_experiment_field',
+		'gutenberg-experiments',
+		'gutenberg_experiments_section',
+		array(
+			'label' => __( 'Test partial syncing of patterns', 'gutenberg' ),
```

Comment:
I think we could add a better description to this, but maybe I can do a follow-up PR that adds some more detail.